src/components/timeline/TrackEventDelay.js

```diff
@@ -5,15 +5,15 @@
 // @flow
 
 import * as React from 'react';
-import { TrackEventDelayGraph } from './TrackEventDelayGraph';
+import { TrackEventDelayGraph } from 'firefox-profiler/components/timeline/TrackEventDelayGraph';
```

Comment:
Please revert this line, it starts with `./` and it's not related to `src/actions`.